modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelResourceFactoryTestCase.java

```diff
@@ -6,24 +6,29 @@
  */
 package org.mule.runtime.module.extension.internal.resources;
 
+import static java.lang.Thread.currentThread;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.junit.Assert.assertThat;
 import static org.mule.runtime.module.extension.api.util.MuleExtensionUtils.loadExtension;
 import static org.mule.runtime.module.extension.internal.ExtensionProperties.EXTENSION_MODEL_JSON_FILE_NAME;
 import org.mule.runtime.api.meta.model.ExtensionModel;
+import org.mule.runtime.core.api.util.IOUtils;
 import org.mule.runtime.extension.api.persistence.ExtensionModelJsonSerializer;
 import org.mule.runtime.extension.api.resources.GeneratedResource;
 import org.mule.runtime.extension.api.resources.spi.GeneratedResourceFactory;
+import org.mule.runtime.module.extension.internal.resources.documentation.ExtensionDocumentationResourceGenerator;
 import org.mule.tck.size.SmallTest;
 import org.mule.test.heisenberg.extension.HeisenbergExtension;
 
+import org.custommonkey.xmlunit.XMLUnit;
 import org.junit.Before;
 import org.junit.Test;
 
 @SmallTest
 public class ExtensionModelResourceFactoryTestCase extends AbstractGeneratedResourceFactoryTestCase {
```

Comment:
class name should change too

Reply:
I don't agree, all extension resource factories should be tested here, we currently have only one, but this could change